### PR TITLE
Update digital-questions.md

### PR DESCRIPTION
**Pull request by Rupesh1510**

### CODE REVIEW

**Comment by Rupesh1510** [comment]
This is a verified question that was asked during Synopsy's technical test.

**Comment by mikinty** [comment]
Thanks a lot! Glad people are contributing to this :)

File: questions/digital-questions.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
   * Fully encoded
 * Explain the difference between a Mealy and Moore machine
 * Design a latch
+* Desing XOR gate using 2:1 MUX [solution](https://vlsiuniverse.blogspot.com/2016/12/XOR-gate-using-mux.html)
```
Comment (mikinty) [inline questions/digital-questions.md]: Can you fix the typo? Should be "Design"

**Review by mikinty — CHANGES_REQUESTED** [review]

File: questions/digital-questions.md (hunk 1 above)
Comment (Rupesh1510) [reply questions/digital-questions.md]: Thank you for pointing that out. I have corrected it. it is my first pull request. I am a student learning to use GitHub. Probably I'll contribute to this repo more.

**Review by mikinty — APPROVED** [review]

**Comment by mikinty** [comment]
Thanks for your updates

**Pull request merged**